.travis.yml

```diff
@@ -32,13 +32,7 @@ matrix:
   include:
     - env: TOXENV=flake8
     - env: TOXENV=docs
-    - env: TOXENV=py27-core
-    - env: TOXENV=py27-dropbox
-      if: env(DROPBOX_APP_TOKEN) is present
-    - env: TOXENV=py27-unixsocket
-    - env: TOXENV=py27-apache
-    - env: TOXENV=py27-aws
-    - env: TOXENV=py27-postgres
```

Comment:
i don't see a py3.x version of postgres tests

Reply:
@dlstadther 
i think that the first thing we should merge into py2-deprecation is https://github.com/spotify/luigi/pull/2836/ because all of this type issues (tox, travis) are fixed there
Just for not rewriting the same things in the following prs like this one

Reply:
@dlstadther 
anyway did it